src/lib.rs

```diff
@@ -94,6 +96,8 @@ macro_rules! register_module {
                 // Suppress the default Rust panic hook, which prints diagnostics to stderr.
                 ::std::panic::set_hook(::std::boxed::Box::new(|_| { }));
 
+                // During tests, node is not available. Skip module registration.
+                #[cfg(not(test))]
```

Comment:
Is it worth adding a crate feature to allow forcing this even when building tests? I can't think of a use case.

Reply:
I'd say wait until we know it's an issue.